src/display/api.js

```diff
@@ -75,7 +75,7 @@ if (typeof PDFJSDev !== 'undefined' && PDFJSDev.test('GENERIC')) {
         worker = __non_webpack_require__('./pdf.worker.js');
       }
       callback(worker.WorkerMessageHandler);
-    });
+    }, null, 'pdfWorker');
```

Comment:
`pdfjsWorker` seems like a slightly better choice here, since it would be consistent with the name/alias [used in the bundles](https://github.com/mozilla/pdf.js/blob/436d2efa8a89edef3a05f70592e3a70fd0c7d11e/gulpfile.js#L288).

Reply:
thanks, updated!